### PR TITLE
Ignore expected warning in LIS install Scenarios

### PR DESCRIPTION
**Pull request by sireeshapotnuru**
This PR meets below requirements

1) As part of fixing Depmod issue, the following warning is an expected one. Hence handled a check in the code to ignore this
2) Check whether pci_hyperv is loaded for Distro above 7.3, since it is not supported for < 7.3

### CODE REVIEW

**Comment by sireeshapotnuru** [comment]
[LISAv2 Test Results Summary]
Test Run On           : 05/20/2019 11:02:51
ARM Image Under Test  : RedHat : RHEL : 7.3 : latest
Test Category         : Functional
Test Area             : LIS_DEPLOY
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:22

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 LIS_DEPLOY           LIS-DEPLOY-SCENARIO-3                                                             PASS                 16.4 
	Install-LIS-Scenario-3 : PASS 


Even the warning is displayed, test has ignored it 

05/20/2019 11:20:28 : [INFO ] Uninstalling the Linux Integration Services for Microsoft Hyper-V...
Removing Package..
Stopping KVP Daemon....
Stopping FCOPY Daemon....
Stopping VSS Daemon....
Create and install initramfs without Hyper-V drivers
Linux Integration Services for Hyper-V has been removed.
>> warning: /etc/depmod.d/hyperv.conf saved as /etc/depmod.d/hyperv.conf.rpmsave
Uninstalled Linux Integration Services for Microsoft Hyper-V. Please reboot your system.

**Comment by iamshital** [comment]
@sireeshapotnuru Let's check if we can make a generic XML for this.

**Review by lizzha — APPROVED** [review]

File: Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh [hunk 1]
```diff
@@ -69,7 +69,7 @@ isLISInstalled=$(rpm -qa | grep hyper-v 2>/dev/null)
 
 if [ ! -z "$isLISInstalled" ]; then
     expected_lis_version=$(dmesg | grep 'Vmbus LIS version' | awk -F ':' '{print $3}' | tr -d [:blank:])
-    if [[ $DISTRO_VERSION =~ 7\. ]]; then
+    if version_ge $DISTRO_VERSION "7.3" ; then
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh]: just found that function name 'version_ge'. It does not make sense of the name. Is it truncated or meant 'greater than or equal'? Also that function definition part should be revised. The comment along with this function does not make sense.

File: Testscripts/Linux/LIS-MODULES-CHECK.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def RunTest(command):
     [current_distro, distro_version] = DetectDistro()
     lis_exists=Run("rpm -qa | grep hyper-v 2>/dev/null")
 
-    if distro_version.startswith("7") and lis_exists:
+    if LooseVersion(distro_version) >= LooseVersion('7.3') and lis_exists:
```
Comment (juhlee-microsoft) [inline Testscripts/Linux/LIS-MODULES-CHECK.py]: Let's define the supported distro version as global constant and use it inside code.

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

File: Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh (hunk 1 above)
Comment (sireeshapotnuru) [reply Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh]: Here ge means 'greater than or equal' 
we are utilizing this function for checking if the argument 1 >= argument 2 
will update the function description accordingly

Comment (juhlee-microsoft) [reply Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh]: Thanks. look forward to it.

Comment (sireeshapotnuru) [reply Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh]: Updated the description of function

File: Testscripts/Linux/LIS-MODULES-CHECK.py (hunk 2 above)
Comment (sireeshapotnuru) [reply Testscripts/Linux/LIS-MODULES-CHECK.py]: assigned distro version to a variable and used inside code

**Comment by sireeshapotnuru** [comment]
> @sireeshapotnuru Let's check if we can make a generic XML for this.

@iamshital created a file XML/Other/ignorable-test-warnings.xml, where it has list of warnings that can  be ignored

**Review by iamshital — APPROVED** [review]

**Comment by sireeshapotnuru** [comment]
@juhlee-microsoft  Please approve and merge this PR

File: Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh (hunk 1 above)
Comment (juhlee-microsoft) [reply Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh]: Can you post the commit # for this change? I don't see it in GitHub UI.

Comment (sireeshapotnuru) [reply Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh]: i modified in this commit only .. below is the updated description for the function version_ge
-->Check if version is greater than or equal to supported version

Comment (sireeshapotnuru) [reply Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh]: @juhlee-microsoft Changed function name to check_version_greater_equal from version_ge

**Review by juhlee-microsoft — APPROVED** [review]

**Pull request merged**